test/updateItem.js

```diff
@@ -1450,6 +1450,11 @@ describe('updateItem', function() {
         UpdateExpression: 'SET #b = :b',
         ExpressionAttributeNames: {'#a': 'active', '#b': 'active'},
         ExpressionAttributeValues: {':a': {BOOL: false}, ':b': {BOOL: true}},
+      }, {
+        ConditionExpression: '#a IN (:a)',
+        UpdateExpression: 'SET #a = :b',
+        ExpressionAttributeNames: {'#a': 'active'},
+        ExpressionAttributeValues: {':a': {BOOL: false}, ':b': {BOOL: true}},
```

Comment:
To make test cases comprehensive, consider adding all kinds of JS falsy things: `false, null, 0, 0.0, ""`

Reply:
Added another test for those values. `null` and `0`/`0.0` don't fail because they're encoded as `{NULL:true}` and `{N: "0"}` respectively, so their "attrVal" becomes `true` and `"0"`. 

The one that confuses me as to how it didn't fail is `""` (`{S:""}`) because I feel like that should have the same issue but doesn't for reasons I cannot understand.